source/base/data_out_base.cc

```diff
@@ -5345,7 +5345,7 @@ namespace DataOutBase
         // underscores unless a vector name has been specified
         out << "VECTORS ";
 
-        if (std::get<2>(nonscalar_data_range) != "")
+        if (!std::get<2>(nonscalar_data_range).empty())
```

Comment:
Thanks for editing these too. I don't mind including them in the PR as long as they stay in a separate commit.